app/src/main/java/com/example/navadon/androidonclickevent/Option0Activity.java

```diff
@@ -27,6 +28,8 @@ private void bindView() {
         etInput = findViewById(R.id.et_input_0);
         tvOutput = findViewById(R.id.tv_body_0);
         btnProcess = findViewById(R.id.btn_process_0);
+        next = findViewById(R.id.next);
```

Comment:
It would be better to name the variable like "nextButton" which is easier to understand.